src/java/org/apache/cassandra/cql3/functions/masking/ColumnMask.java

```diff
@@ -93,6 +94,14 @@ public List<AbstractType<?>> partialArgumentTypes()
                : argTypes.subList(1, argTypes.size());
     }
 
+    /**
+     * @return The values of the arguments of the partially applied masking function.
+     */
+    public List<ByteBuffer> partialArgumentValues()
+    {
+        return Arrays.asList(partialArgumentValues);
```

Comment:
We probably want to protect that List against modifications.

Reply:
Done

Reply:
Actually not properly done, I was using Guava's immutable list, that doesn't support null arguments. Changed to Java's `Collections.unmodifiableList`, which perfectly supports null arguments.

Reply:
^does this deserve a test/comment? The 3 of us missed it in the first instance

Reply:
Not sure since it's pretty obvious and tests detect it immediately, but added a comment anyway.

Reply:
I was more worried about tests catching it than anything else. Sgtm